### PR TITLE
Simple doc content edits - section 1 -5

### PR DESCRIPTION
**Pull request by mosabua**
A whole bunch minor/trivial changes done while reading through the whole docs to learn.

- comma usage
- removal of future and would/could as possible
- grammar and spelling

Sections 1 to 5

- overview 
- installation
- security
- admin
- optimizer

Signed-off-by: Manfred Moser <mmoser@apache.org>
Signed-off-by: Manfred Moser <manfred@simpligility.com>

Follow up for connectors section and others is separate to reduce PR size. See https://github.com/prestosql/presto/pull/1964

### CODE REVIEW

**Comment by mosabua** [comment]
Do not review or merge yet. Going through the complete docs...

File: presto-docs/src/main/sphinx/security/ldap.rst [hunk 1]
```diff
@@ -204,7 +204,7 @@ Example:
 For OpenLDAP, for this query to work, make sure you enable the
 ``memberOf`` `overlay <http://www.openldap.org/doc/admin24/overlays.html>`_.
 
-You can also use this property for scenarios where you want to authorize a user
+You can  use this property for scenarios where you want to authorize a user
```
Comment (findepi) [inline presto-docs/src/main/sphinx/security/ldap.rst]: stuttered space

**Comment by findepi** [comment]
I like these changes!

BTW The repeating part is change the future tense into present -- the future is now 😉

File: presto-docs/src/main/sphinx/security/ldap.rst (hunk 1 above)
Comment (mosabua) [reply presto-docs/src/main/sphinx/security/ldap.rst]: fixed

**Comment by mosabua** [comment]
@mattsfuller @findepi  ... I can squash and merge myself once you give me committer access .. otherwise whoever wants to proceed this is ready to go.

**Comment by findepi** [comment]
Merged as eff147f51e0d610c2b9bcab6bcb9dfaa1d37e1ee, thanks!

(what i meant is that you could squash and force push the branch, so that merge can happen via GH UI)

**Comment by mosabua** [comment]
oh .. I did not realize you wanted me to force push .. I generally avoid doing that whenever I can .. I think it might be worth for me to figure out what the standard git and github usage is ...

**Pull request closed without merging**